virtualization/windowscontainers/kubernetes/getting-started-kubernetes-windows.md

```diff
@@ -1,152 +1,192 @@
+
+```powershell
+Install-Module -Name DockerMsftProvider -Repository PSGallery -Force
+Install-Package -Name Docker -ProviderName DockerMsftProvider
+Restart-Computer -Force
+```
+
+If you are behind a proxy, the following PowerShell environment variables must be defined:
+```powershell
+[Environment]::SetEnvironmentVariable("HTTP_PROXY", "http://proxy.example.com:80/", [EnvironmentVariableTarget]::Machine)
+[Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
+```
+
+There is a collection of scripts on [this Microsoft repository](https://github.com/Microsoft/SDN) that will help us join this node to the cluster. You can download the ZIP file directly [here](https://github.com/Microsoft/SDN/archive/master.zip). The only thing we need is the `Kubernetes/windows` folder, the contents of which should be moved to `C:\k\`:
+
+```powershell
+wget https://github.com/Microsoft/SDN/archive/master.zip -o master.zip
+Expand-Archive master.zip -DestinationPath master
+mkdir C:/k/
+mv master/SDN-master/Kubernetes/windows/* C:/k/
+rm -recurse -force master,master.zip
+```
+
+Copy the certificate file [identified earlier](#preparing-the-linux-master) to this new `C:\k` directory.
+
+
+### Creating the "Pause" Image ###
+Now that `docker` is installed, we need to prepare a "pause" image that's used by Kubernetes to prepare the infrastructure pods.
+
+```powershell
+docker pull microsoft/windowsservercore:1709
+docker tag microsoft/windowsservercore:1709 microsoft/windowsservercore:latest
+cd C:/k/
+docker build -t kubeletwin/pause .
+```
+
+> [!Note]  
+> We tag it as the `:latest` because the sample service we will be deploying later depends on it, though this may not actually _be_ the latest Windows Server Core image available. It's important to be careful of conflicting container images; not having the expected tag can cause a `docker pull` of an incompatible container image, causing [deployment problems](./common-problems.md#when-deploying-docker-containers-keep-restarting). 
+
+
+### Downloading Binaries ###
+In the meantime while the `pull` occurs, download the following client-side binaries from Kubernetes:
+
+  - `kubectl.exe`
+  - `kubelet.exe`
+  - `kube-proxy.exe`
+
+You can download these from the links in the `CHANGELOG.md` file of the latest 1.9 release. As of this writing, that is [1.9.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.9.1), and the Windows binaries are [here](https://storage.googleapis.com/kubernetes-release/release/v1.9.1/kubernetes-node-windows-amd64.tar.gz). Use a tool like [7-Zip](http://www.7-zip.org/) to extract the archive and place the binaries in `C:\k\`.
+
+In order to make kubectl command available outside of `C:\k\` directory, modify PATH environment variable:
+```powershell
+$env:Path = $env:Path + ";C:\k"
+```
```

Comment:
Super nitpick: "...make **the** `kubectl` command available..."

Also, the script block can use the `+=` operator.